lib/improver/units.py

```diff
@@ -62,10 +62,55 @@
     "projection_y_coordinate": {"unit": "m"},
     "percentile": {"unit": "%"},
     "probability": {"unit": "1"},
+    "realization": {"unit": "1", "dtype": np.int32},
+    "height": {"unit": "m"},
+    "velocity": {"unit": "m s-1"},
```

Comment:
Diagnostic rather than coordinate?

Reply:
Velocity moved to diagnostics.

Reply:
I also alphabetised the rest of the sections.